src/DiseaseStatus.H

```diff
@@ -168,42 +156,38 @@ void DiseaseStatus<AC,ACT,ACTD,A>::updateAgents(AC& a_agents, /*!< Agent contain
                     }
                     else if (status_ptr[i] == Status::infected) {
                         counter_ptr[i] += 1;
-                        if (counter_ptr[i] == 1) {
+                        if (counter_ptr[i] == 1) { // just infected
                             if (Random(engine) < disease_parm_d->p_asymp[0]) {
                                 symptomatic_ptr[i] = SymptomStatus::asymptomatic;
                             } else {
                                 symptomatic_ptr[i] = SymptomStatus::presymptomatic;
                             }
                         }
                         if (counter_ptr[i] == Math::floor(incubation_period_ptr[i])) {
+                            // symptoms can now appear, if non-asynmptomatic
                             if (symptomatic_ptr[i] != SymptomStatus::asymptomatic) {
+                                AMREX_ASSERT(symptomatic_ptr[i] == SymptomStatus::presymptomatic);
                                 symptomatic_ptr[i] = SymptomStatus::symptomatic;
-                            }
-                            if (    (symptomatic_ptr[i] == SymptomStatus::symptomatic)
-                                &&  (symptomatic_withdraw)
-                                &&  (Random(engine) < symptomatic_withdraw_compliance)) {
-                                withdrawn_ptr[i] = 1;
-                            }
-
-                            if (symptomatic_ptr[i] == SymptomStatus::symptomatic) {
-                                int flag_ICU_i = 0, flag_vent_i = 0;
+                                // could be hospitalized when symptoms first appear
                                 Real num_days = 0;
                                 disease_parm_d->check_hospitalization( num_days,
-                                                                       flag_ICU_i,
-                                                                       flag_vent_i,
+                                                                       flag_ICU_ptr[i],
+                                                                       flag_vent_ptr[i],
                                                                        age_group_ptr[i],
-                                                                       u50frac,
                                                                        engine );
                                 timer_ptr[i] = ParticleReal(num_days);
-                                if (timer_ptr[i] > 0) { flag_hosp_ptr[i] = 1; }
-                                if (flag_ICU_i) { flag_ICU_ptr[i] = 1; }
-                                if (flag_vent_i) { flag_vent_ptr[i] = 1; }
+                                if (timer_ptr[i] > 0) flag_hosp_ptr[i] = 1;
```

Comment:
I think retaining the `{ ... }` even if the `if` condition has only one line is an AMReX practice. Can you check with @atmyers?

Reply:
According to the AMReX style [guide](https://github.com/AMReX-Codes/amrex/blob/development/CONTRIBUTING.md#amrex-coding-style-guide), that only applies to `for` loops. I do think it would be a good idea to use something like clang-format with an agreed upon style for consistency across all the contributors. We should discuss as a group.